Mehdi_test/date_f.nit

```diff
@@ -0,0 +1,104 @@
+#Module date_f has the basic date fuction implementation
+module date_f
+
+#The main class Date
+public class Date
+
+private var years_In : Int 
+private var mounth_In : Int
+private var day_In : Int
+
+private var hours_In : nullable Int = 0
+private var min_In : nullable Int = 0
+private var sec_In : nullable Int = 0
+
+	#Return the with String type
+	redef fun to_s do
+
+	var y = years_In.to_s
+	var m = mounth_In.to_s
+	var d = day_In.to_s
+
+	return y +"/"+ m +"/"+ d
```

Comment:
You should probably use a formatting string `"{y}/{m}/{d}", it is a bit easier to read.